### DISCUSSION TITLE
As a user, I want to validate multiple bundles at once

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/validate#796] [Q&A] [answered]
When I try to validate 2 bundles at once, validate doesn't figure it out. What gives?

**Comment by pds-ops** [comment, marked as answer]
This is actually a new requirement for validate that has not been implemented here #237.

In the meantime, there are a couple possible workarounds for this:

### Validate each bundle in parallel
This is probably the most efficient solution by running 2 validate executions at the same time, by running each validate execution in the background. There are numerous ways you could do this [per this online documentation.](https://www.makeuseof.com/run-linux-commands-in-background/).

### Validate each bundle via 1 line command
(in bash)
```
validate --rule pds4.bundle --target /path/to/bundle_1.xml ; \
validate --rule pds4.bundle --target /path/to/bundle_2.xml
```
